docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,255 @@
+
+## Create A Project
+
+With our tools and documentation installed and open, let’s start by creating a new Android Studio project.  It could be any
+type of project to start with, but I’m choosing the “Basic Activity” template simply because it comes with a button already on
+the screen that can use to crash our app in order to collect our metrics.  Give the project a name of "GleanCrashExample",
+choose Kotlin for the language, and choose API level 21 for the minimum SDK level.
+
+## Setup Build Configuration
+
+### Add Glean As A Dependency
+
+Now that we have a project started, we can add Glean to it as a dependency in the Gradle configuration.  Find the
+`build.gradle` file for your _app module_ and add the following line to the dependencies section:
+
+```Groovy
+implementation 'org.mozilla.components:service-glean:16.0.0'
+```
+
+As of this writing, version `16.0.0` is the latest version of [Android Components](https://github.com/mozilla-mobile/android-components/).  
+Android Components is a collection of tools for building browsers on Android and it is the easiest way to consume Glean on
+Android at the moment.  You should probably take a look at the Android-Components releases and check that you are using the
+latest version of Android-Components for your project.  So just replace the version above with the latest.
+
+### Add Mozilla Maven Repository
+
+In order to be able to download Glean, we need to add Mozilla’s maven repository URL to the Project `build.gradle` file.  In
+Android Studio, for our project named “GleanCrashExample”, there will be a `build.gradle` that has `(Project: GleanCrashExample)`
+next to it.  In this Gradle file, you will find a section called `allProjects` which has in it a section called `repositories`.
+You may already find `google()` and `jcenter()` there, but we need to add the following in order to download Glean:
+
+```Groovy
+maven {
+    url "https://maven.mozilla.org/maven2"
+}
+```
+
+### Add Python Environment
+
+We add a Python environment for the [glean_parser](https://github.com/mozilla/glean_parser/) to run in with the following lines,
+added to the very top of the `build.gradle` file:
+
+```Groovy
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.26"
+}
+```
+
+### Import SDK Generator Script
+
+The last part of the changes to the app’s `build.gradle` file is to include the `sdk_generator.gradle` script.  Right before the
+end of the `build.gradle` file, add the following (making sure the version here matches the version of the `implementation`
+line above):
+
+```Groovy
+apply from: 'https://github.com/mozilla-mobile/android-components/raw/v16.0.0/components/service/glean/scripts/sdk_generator.gradle'
+```
+
+That should be it for additions to the build configuration files.  You should try to do a Gradle sync now to make sure
+everything works.  If you have any problems, try going back through the configuration steps to make sure you got everything.
```

Comment:
As this will become part of the whole book, would it make sense to link back to https://mozilla.github.io/glean/book/user/adding-glean-to-your-project.html as a "short summary"?

Reply:
(still keeping the section here of course)